src/sentry/static/sentry/app/components/events/contextSummary/contextSummary.tsx

```diff
@@ -13,9 +13,15 @@ import ContextSummaryOS from './contextSummaryOS';
 import ContextSummaryUser from './contextSummaryUser';
 import filterContexts from './filterContexts';
 
+export type Context = {
+  keys: string[];
+  Component: (props: any) => JSX.Element;
```

Comment:
I don't like much the any here, but I think that it's fine for now.. we should refactor this component in the future :)

Reply:
Could we add a `TODO(ts)` here?

Reply:
👍